webpack.config.js

```diff
@@ -77,6 +77,10 @@ const config = {
           },
         },]
       },
+      {
+        test: /\.css$/i,
```

Comment:
This is actually a change required for Cuix components like table. Since I was working on it simultaneously I included it in this commit. Since these actions depend on the table as well moving forward